src/patternfly/components/Banner/examples/Banner.md

```diff
@@ -36,6 +36,64 @@ cssPrefix: pf-c-banner
 {{/banner}}
 ```
 
+### Banner with links
+
+```hbs
+{{#> banner banner--modifier="pf-m-info"}}
+  Banner with a 
+  <a href="https://www.w3.org/TR/WCAG20-TECHS/ARIA8.html#ARIA8-examples">
+  link</a>
+{{/banner}}
+<br>
+{{#> banner banner--modifier="pf-m-info"}}
+  Banner with a 
+  <a class="pf-m-disabled" role="link" aria-disabled="true">
+  disabled link</a>
+{{/banner}}
+<br>
+{{#> banner banner--modifier="pf-m-info"}}
+  Banner with an 
+  {{#> button-link button-link--attribute='href="https://www.w3.org/TR/WCAG20-TECHS/ARIA8.html#ARIA8-examples"' button-link--modifier="pf-m-inline pf-m-link"}}
+  inline link{{/button-link}}
+{{/banner}}
+<br>
+{{#> banner }}
+  Banner with an 
+  {{#> button button--modifier="pf-m-inline pf-m-link"}}
+    inline link button
+  {{/button}}
+{{/banner}}
+<br>
+{{#> banner}}
+  Banner with a 
+  {{#> button button--modifier="pf-m-link pf-m-inline" button--attribute="disabled"}}
+    disabled inline link button
+  {{/button}}
+{{/banner}}
+<br>
+{{#> banner banner--modifier="pf-m-warning"}}
+  A warning banner with 
+  {{#> button button--modifier="pf-m-link pf-m-inline"}}
+    inline link button
+  {{/button}}
+{{/banner}}
+<br>
+{{#> banner banner--modifier="pf-m-danger"}}
+  A long danger banner with both text and {{#> button button--IsInlineLinkSpan="true" button--attribute='tabindex="0"' button--modifier="pf-m-link pf-m-inline"}}
+    long button text that needs to be a span so that it will wrap inline with the text around it.
+  {{/button}}
+{{/banner}}
+<br>
+{{#> banner banner--modifier="pf-m-info"}}
```

Comment:
looks like @mcarrano wants to remove this example

Reply:
@mcoker @mcarrano Do we actually need most of these examples? I wanted to make sure they all worked, but should we simply add the one preferred method (which one?) of using a link within the banner into one or all of the existing examples?

Reply:
@srambach @mcoker I would be in favor of that. Maybe just update the existing examples to include a link in the message text. That way you would see what the link looks like for each color background.